pyUltroid/fns/openaihelper.py

```diff
@@ -0,0 +1,239 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~# SOL UserBot #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+# sqlalchemy
+# openai
+# fake_useragent
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+# Special Credits: @LucifSD02 <Telegram>
```

Comment:
again, commit attributes exist, why populate code with credits?
(also, missing copyright header)a

Reply:
i removed this file as in not needed